README.md

```diff
@@ -0,0 +1,34 @@
+# BETTER BUILDS
```

Comment:
for readability, let's add a new empty line before and after headers (don't need an empty space before the first heading), should look like:
```md
# Better Builds

content here

## Contributors

content here
```